src/resource/resource-opening-hours/opening-period/PeriodOpeningHours.test.tsx

```diff
@@ -95,6 +95,93 @@ const testDatePeriod: DatePeriod = {
   ],
 };
 
+const testDatePeriodWithMissingStartAndEndTimes: DatePeriod = {
+  id: 1,
+  created: '2020-11-20',
+  modified: '2020-11-20',
+  is_removed: false,
+  name: { fi: '', sv: '', en: '' },
+  description: { fi: 'Testikuvaus', sv: '', en: '' },
+  start_date: '',
+  end_date: '2020-11-21',
+  resource_state: ResourceState.OPEN,
+  override: false,
+  resource: 1,
+  time_span_groups: [
+    {
+      id: 5807,
+      period: 6937,
+      time_spans: [
+        {
+          id: 9054,
+          group: 5807,
+          name: {
+            fi: null,
+            sv: null,
+            en: null,
+          },
+          description: {
+            fi: 'Aukiolon testikuvaus',
+            sv: null,
+            en: null,
+          },
+          start_time: '',
```

Comment:
Does API return also empty strings?

Reply:
Actually it probably does not, so I'll change this to null:

- [ ] Change empty string to null

Reply:
Ok, I thought there was something, hopefully we are right 😬 😁